etc/dino.profile

```diff
@@ -5,6 +5,7 @@ include /etc/firejail/dino.local
 # Persistent global definitions
 include /etc/firejail/globals.local
 
+noblacklist ${DOWNLOADS}
```

Comment:
Hey @SpotComms :-) Are you having plans for the corresponding `blacklist ${DOWNLOADS}`?